### PR TITLE
List Sunplatform (SUNP)

### PR DESCRIPTION
**Pull request by sunplatformCore**
- Official project URL: https://github.com/sunplatformCore/sunplatform
- Official block explorer URL: http://45.77.13.137/

### CODE REVIEW

File: assets/src/test/java/bisq/asset/coins/SUNPTest.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        assertValidAddress("SQEasE1ErGeSSqjruzL879UNPeV7sWbnAN");
+        assertValidAddress("SRK1eahAugt3k6BQjbR1EFYWTAGzg6uXPd");
+        assertValidAddress("SfCe8XrgG2FTPLyuheJ2kMVTD4QehjPUPb");
+        assertValidAddress("SkSJUTCgfJyGM6REhvt54JNQswnwsGHt5E");
+    }
+
+    @Test
+    public void testInvalidAddresses() {
+        assertInvalidAddress("sQEasE1ErGeSSqjruzL879UNPeV7sWbnAN");
+        assertInvalidAddress("SRK1eahBugt3k6BQjbR1EFYWTAGzg6uXpD");
```
Comment (CaveSpectre11) [inline assets/src/test/java/bisq/asset/coins/SUNPTest.java:41]: The middle 3 addresses are valid according to your regex

**Review by CaveSpectre11 — CHANGES_REQUESTED** [review]
3 of the Invalid Addresses match the regex; as valid addresses:

```
$ egrep -v -e '.*valid [S][a-km-zA-HJ-NP-Z1-9]{24,33}$' addresses.txt
invalid sQEasE1ErGeSSqjruzL879UNPeV7sWbnAN
invalid SfCe8XrgG2

 $ egrep -e '.*valid [S][a-km-zA-HJ-NP-Z1-9]{24,33}$' addresses.txt
valid SQEasE1ErGeSSqjruzL879UNPeV7sWbnAN
valid SRK1eahAugt3k6BQjbR1EFYWTAGzg6uXPd
valid SfCe8XrgG2FTPLyuheJ2kMVTD4QehjPUPb
valid SkSJUTCgfJyGM6REhvt54JNQswnwsGHt5E
invalid SRK1eahBugt3k6BQjbR1EFYWTAGzg6uXpD
invalid SfCe8XrgG2FTPLyuheJ2kMVTD4QehjLULd
invalid SSSJUTCgfJyGM6REhvt54JNQswnwsGHt6e

```

**Review by blabno — CHANGES_REQUESTED** [review]
NACK
Do not submit code that you have not tested locally!

**Comment by sunplatformCore** [comment]
I tested in windows it said build succeeded
I run gradlew.bat

**Pull request closed without merging**